src/main/java/duke/command/DukeException.java

```diff
@@ -0,0 +1,25 @@
+package duke.command;
+
+/**
+ * Handles any command that is not understandable by Duke.
+ */
+public class DukeException extends Exception {
+    private final String _message = " OOPS!!! I'm sorry, but I don't know what that means :-(";
+    private String _error;
```

Comment:
You should avoid using _ in variable names.
https://www.geeksforgeeks.org/using-_-underscore-variable-name-java/